### ISSUE TITLE
Mounting doesn't work

### ISSUE THREAD

**Issue by OrangeSucculenc** [vKolerts/sidenoder#24]
I'm kinda new to this and don't know if mounting is even necessary, but whenever I launch the program it shows this error, but I'm sure I have done everything correctly.
![image](https://user-images.githubusercontent.com/82822337/149978958-808347fc-7c4e-4cf6-9f03-342abbf3bb5c.png)

**Comment by ShibbyBro** [comment]
Having the same issue on a mac. None of the mirrors will moun
![Screen Shot 2022-01-18 at 3 07 21 PM](https://user-images.githubusercontent.com/22160019/150019116-ad4548fc-fcc4-488b-8db2-431a5dacaa0c.png)
t

**Comment by Liace** [comment]
> I'm kinda new to this and don't know if mounting is even necessary, but whenever I launch the program it shows this error, but I'm sure I have done everything correctly. ![image](https://user-images.githubusercontent.com/82822337/149978958-808347fc-7c4e-4cf6-9f03-342abbf3bb5c.png)



> Having the same issue on a mac. None of the mirrors will moun ![Screen Shot 2022-01-18 at 3 07 21 PM](https://user-images.githubusercontent.com/22160019/150019116-ad4548fc-fcc4-488b-8db2-431a5dacaa0c.png) t

Okay,
You need to update the rclone binaries for that, Kolerts must have forgoten to pack the new version with it. You can patch it by simply downloading the new version and dropping and replacing the files in "C:\Users\[Username]\sidenoder".

Hope you have nice new VR game sessions !  

EDIT: Hum.. While yes it solved the error and mounted properly, I found a bigger problem right after.. VRPirates haved changed their way of sharing official oculus catalog and request an update for their soft.. Big oof. 
We'll have to wait a new sidenoder update I guess.

**Comment by KaladinDMP** [comment]
> EDIT: Hum.. While yes it solved the error and mounted properly, I found a bigger problem right after.. VRPirates haved changed their way of sharing official oculus catalog and request an update for their soft.. Big oof. We'll have to wait a new sidenoder update I guess.

You are correct but easier to fix by launching a command window in the current rclone.exe directory and running "rclone selfupdate" 

That updates the binary without having to do anything else which supports mounting.  About your other issue, you can get the new config right now at the [Sidenoder Telegram](t.me/sidenoder) 
 This allows you to use FFA mirrors as well as VRPirates and their mirrors actually work...

**Comment by Liace** [comment]
> > EDIT: Hum.. While yes it solved the error and mounted properly, I found a bigger problem right after.. VRPirates haved changed their way of sharing official oculus catalog and request an update for their soft.. Big oof. We'll have to wait a new sidenoder update I guess.
> 
> You are correct but easier to fix by launching a command window in the current rclone.exe directory and running "rclone selfupdate"
> 
> That updates the binary without having to do anything else which supports mounting. About your other issue, you can get the new config right now at the [Sidenoder Telegram](t.me/sidenoder) This allows you to use FFA mirrors as well as VRPirates and their mirrors actually work...

Ho Thx for the precisions! But the link for the telegram Sidenoder channel doesn't work, I would be glad to be able to more closely follow Sidenoder news but I can't find the link elsewhere..
Is the solution you're talking about just using resilio and pointing rclone to the local placeholder files it downloaded?

**Comment by KaladinDMP** [comment]
Sorry I don't know how I messed up the the link it's t.me/sidenoder.


Has nothing to do with resilio no. It's to use the program as intended, resilio is crap, VRP only uses it because they can't figure out their mirror issue.s This is to give sidenoder FFA mirrors that work

**Comment by vKolerts** [comment]
Please try this solution https://github.com/vKolerts/sidenoder/issues/25#issuecomment-1018618845

**Comment by baza64** [comment]
![Screenshot_20220207_170422](https://user-images.githubusercontent.com/99193679/152806668-7449188e-5ef2-42f8-942b-d48b62cb3b19.png)
 Whats wrong? help me pls

Linux Manjaro

**Comment by vKolerts** [comment]
mirror expired. Check other mirrors or wait for update of rclone configs at tg channel

**Comment by baza64** [comment]
thx;)))))%)15:06, 16 февраля 2022 г., vKolerts ***@***.***>:
mirror expired. Check other mirrors or wait for update of rclone configs at tg channel

—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you commented.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс.Почты
